docs/curate-text/process-data/quality-assessment/classifier.md

```diff
@@ -164,32 +164,31 @@ Use the pre-trained `nvidia/quality-classifier-deberta` model for robust quality
 from nemo_curator.pipeline import Pipeline
 from nemo_curator.stages.text.io.reader import JsonlReader
 from nemo_curator.stages.text.io.writer import JsonlWriter
-from nemo_curator.stages.text.modules import DistributedDataClassifier
 from nemo_curator.stages.text.classifiers import QualityClassifier
 
 # Create pipeline with DeBERTa quality classifier
 pipeline = Pipeline(name="quality_classification_pipeline")
 
-# Add stages
-read_stage = JsonlReader("input_data/")
-classify_stage = DistributedDataClassifier(
-    QualityClassifier(
-        model_path="nvidia/quality-classifier-deberta",
-        labels=["Low", "Medium", "High"],  # Quality levels
-        batch_size=256,
-        max_chars=2000
-    ),
-    text_field="text",
-    pred_column="quality_pred"
+# Load dataset
+read_stage = JsonlReader(
+    file_paths="input_data/*.jsonl",
+    fields=["text", "id"]
 )
-write_stage = JsonlWriter("classified_output/")
-
 pipeline.add_stage(read_stage)
+
+# Apply quality classifier
+classify_stage = QualityClassifier(
+    max_chars=6000,  # Default is 6000
+    model_inference_batch_size=256
+)
 pipeline.add_stage(classify_stage)
+
+# Save results
+write_stage = JsonlWriter(path="classified_output/")
 pipeline.add_stage(write_stage)
 
 # Execute pipeline
-results = pipeline.run()
+results = pipeline.run()  # Uses XennaExecutor by default
```

Comment:
let's remove the comment unless we are going to mention this everywhere where pipeline.run is being called
```suggestion
results = pipeline.run() 
```